src/WinUIEx/MonitorInfo.cs

```diff
@@ -32,10 +33,34 @@ public unsafe static IList<MonitorInfo> GetDisplayMonitors()
             return list;
         }
 
+        /// <summary>
+        /// Gets the display monitor that is nearest to a given window.
+        /// </summary>
+        /// <param name="hwnd">Window handle</param>
+        /// <returns>The display monitor that is nearest to a given window.</returns>
+        public unsafe static MonitorInfo GetWindowDisplayMonitor(IntPtr hwnd)
+        {
+            var windowMonitor = PInvoke.MonitorFromWindow(new(hwnd), MONITOR_FROM_FLAGS.MONITOR_DEFAULTTONEAREST);
+            MonitorInfo monitorInfo = null!;
```

Comment:
Why the null-forgiveness? It seems to me this method could end up returning null if the condition is false.